src/bindings/python/src/openvino/frontend/pytorch/decoder.py

```diff
@@ -331,10 +331,56 @@ def mark_node(self, node):
             node.set_friendly_name(name)
         return node
 
+    @staticmethod
+    def convert_quantized_tensor(qtensor: torch.Tensor):
```

Comment:
Is it `dequantize` method? Is it how PyTorch handles such tensors

Reply:
Can you add a test for this case? Are we sure that we need to subtract `zero_point` instead of summation?